shoov/drupal-org.make

```diff
@@ -113,14 +113,14 @@ projects[views][subdir] = "contrib"
 projects[views][version] = "3.11"
 
 projects[views_bulk_operations][subdir] = "contrib"
-projects[views_bulk_operations][version] = "3.2"
+projects[views_bulk_operations][version] = "3.3"
 
 ; Development
 projects[devel][subdir] = "development"
 projects[devel][version] = "1.5"
 
 projects[migrate][subdir] = "development"
-projects[migrate][version] = 2.5
+projects[migrate][version] = 2.8
```

Comment:
I installed on my local. I see an error ``Passing an empty first parameter to a migration constructor is now deprecated - pass through the arguments array passed to the leaf class instead.`` but all data is migrated. So I think I'll change the version back for now.